containers/podman-desktop
# feat: retrieve proxy settings from system

Fixes #3769
Fixes #4765

### What does this PR do?

Get the proxy settings from the system. On Windows it uses the registry. On MacOS networksetup tool is used. On Linux HTTP_PROXY, HTTPS_PROXY and NO_PROXY are used. There are no 3 options for proxy: system (default), manual and disabled.

### Screenshot/screencast of this PR

### What issues does this PR fix or reference?

#3769 
#4765

### How to test this PR?

Set proxy settings on your system and verify there are use if system is used inside Podman Desktop
